### PR TITLE
Linting fixes

### PR DESCRIPTION
**Pull request by josephfusco**
## Description

Squashing minor linter warnings/errors for what is currently in canary.

### CODE REVIEW

**Review by blakewilson — APPROVED** [review]

File: .github/workflows/lint-plugin.yml [hunk 1]
```diff
@@ -4,15 +4,18 @@ jobs:
...
-    - name: Setup PHP
-      uses: shivammathur/setup-php@v2
-      with:
-        php-version: '7.3'
-    - name: Install Dependencies
-      working-directory: plugins/faustwp
-      run: composer install
-    - name: Run Lint
-      working-directory: plugins/faustwp
-      run: composer lint
```
Comment (josephfusco) [inline .github/workflows/lint-plugin.yml:18]: File is now formatted to match `.editorconfig`

File: plugins/faustwp/composer.json [hunk 2]
```diff
@@ -1,37 +1,44 @@
...
-    "scripts": {
-      "lint": "parallel-lint -e php --no-colors --exclude vendor .",
-      "phpcs": "phpcs",
-      "phpcs:fix": "phpcbf",
-      "suite": ["@lint", "@phpcs", "@test"],
-      "test": "phpunit"
-    },
-    "config": {
-      "preferred-install": "dist",
-      "sort-packages": true,
```
Comment (josephfusco) [inline plugins/faustwp/composer.json:34]: File formatting to match `.editorconfig`

File: plugins/faustwp/composer.json [hunk 3]
```diff
@@ -1,37 +1,44 @@
...
+    ],
+    "test": "phpunit"
+  },
+  "config": {
+    "preferred-install": "dist",
+    "sort-packages": true,
+    "optimize-autoloader": true,
+    "allow-plugins": {
+      "dealerdirect/phpcodesniffer-composer-installer": true
     }
```
Comment (josephfusco) [inline plugins/faustwp/composer.json:42]: When running `composer phpcs`, I was getting prompted about [allow-plugins](https://getcomposer.org/doc/06-config.md#allow-plugins) (new security feature in composer 2.2). Declaring our allowed plugins silences the prompt without the need to add the `--no-interaction` flag when running the command.

**Comment by josephfusco** [comment]
@apmatthews Thanks for calling that out! I have added `composer phpcs` to `.github/workflows/lint-plugin.yml` as suggested.

**Pull request merged**